pkg/util/azureclient/graphrbac.go

```diff
@@ -0,0 +1,94 @@
+package azureclient
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/graphrbac/1.6/graphrbac"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+)
+
+const (
+	// https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/api/application_list
+	// To list and patch AAD applications, this code needs to have the clientID
+	// of an application with the following permissions:
+	// API: Windows Azure Active Directory
+	//   Delegated permissions:
+	//      Sign in and read user profile
+	//      Access the directory as the signed-in user
+	clientID = "5935b8e2-3915-409c-bfb2-865b7a9291e0"
```

Comment:
@jim-minter is this correct to have it hardcoded? I wasn't sure if it should be AZURE_CLIENT_ID

Reply:
This should be corresponding to "permission id" so I think its fine.